packages/tabsheet/README.md

```diff
@@ -0,0 +1,64 @@
+# @vaadin/tabsheet
+
+A web component for organizing and grouping content into sections.
+
+[Documentation + Live Demo ↗](https://vaadin.com/docs/latest/components/tabsheet)
+
+[![npm version](https://badgen.net/npm/v/@vaadin/tabsheet)](https://www.npmjs.com/package/@vaadin/tabsheet)
+[![Discord](https://img.shields.io/discord/732335336448852018?label=discord)](https://discord.gg/PHmkCKC)
+
+```html
+<vaadin-tabsheet selected="3">
+  ...
+</vaadin-tabsheet>
```

Comment:
nit: could use a full example, like in the API docs. Not a blocker.

Reply:
Done